### PR TITLE
PEP8 Reformatting

### PR DESCRIPTION
**Pull request by 8bitsam**
Reformatted with black and flake8. The output of `flake8 .` is now just `./diffpy/pdffit2/pdffit.py:151:9: E731 do not assign a lambda expression, use a def`. I didn't want to change this since it would have made the code quite a lot messier in that spot.

### CODE REVIEW

**Comment by sbillinge** [comment]
Thanks @8bitsam . We would need it to pass .flake8 so either paste the lambda in where it is used (preferrable) or define a private function somewhere near the top of the file and then call it there (only if it is very messy the other way)

**Comment by sbillinge** [comment]
@8bitsam we also probably need to fix the conda issue as I don't want to merge anything without tests passing. Do you have a fix for it?

**Comment by 8bitsam** [comment]
> Thanks @8bitsam . We would need it to pass .flake8 so either paste the lambda in where it is used (preferrable) or define a private function somewhere near the top of the file and then call it there (only if it is very messy the other way)

Passes flake8 now, I just used a comment next to the lambda function to ignore the exact issue.

File: diffpy/pdffit2/pdffit.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def intro():
         date = __date__[:10]
         d = {"version": __version__, "date": date, "year": date[:4] or "2019"}
         msg = __intro_message__ % d
-        filler = lambda mx: (mx.group(0).rstrip(" *").ljust(77) + "*")
+        filler = lambda mx: (mx.group(0).rstrip(" *").ljust(77) + "*")  # noqa: E731
```
Comment (sbillinge) [inline diffpy/pdffit2/pdffit.py:151]: I think this lambda needs to be copy-pasted into the row below (and remove the noqa.  To keep it readable you can put a line-break in, so functionally it reads as easily as it does now.

**Review by sbillinge — COMMENTED** [review]
please see comment

File: diffpy/pdffit2/pdffit.py (hunk 1 above)
Comment (sbillinge) [reply diffpy/pdffit2/pdffit.py:151]: btw, I think what you did is fine too, though prefer my solution I think.  But Let's work on getting tests to pass with higher priority....

**Pull request closed without merging**